arcgis-ios-sdk-samples/Display information/Add graphics with renderer/AddGraphicsWithRendererViewController.swift

```diff
@@ -0,0 +1,171 @@
+        // Create a simple marker symbol.
+        let pointSymbol = AGSSimpleMarkerSymbol(style: .diamond, color: .green, size: 10)
+        // Create a graphics overlay for the points.
+        let pointGraphicsOverlay = AGSGraphicsOverlay()
+        // Create and assign a simple renderer to the graphics overlay.
+        pointGraphicsOverlay.renderer = AGSSimpleRenderer(symbol: pointSymbol)
+        return pointGraphicsOverlay
+    }
+    
+    func makeLineGraphicsOverlay() -> AGSGraphicsOverlay {
+        // Create a simple line symbol.
+        let lineSymbol = AGSSimpleLineSymbol(style: .solid, color: .blue, width: 5)
+        // Create a graphics overlay for the polylines.
+        let lineGraphicsOverlay = AGSGraphicsOverlay()
+        // Create and assign a simple renderer to the graphics overlay.
+        lineGraphicsOverlay.renderer = AGSSimpleRenderer(symbol: lineSymbol)
+        return lineGraphicsOverlay
+    }
+    
+    func makePolygonGraphicsOverlay() -> AGSGraphicsOverlay {
+        // Create a simple fill symbol.
+        let polygonSymbol = AGSSimpleFillSymbol(style: .solid, color: .yellow, outline: nil)
+        // Create a graphics overlay for the polygons.
+        let polygonGraphicsOverlay = AGSGraphicsOverlay()
+        // Create and assign a simple renderer to the graphics overlay.
+        polygonGraphicsOverlay.renderer = AGSSimpleRenderer(symbol: polygonSymbol)
+        return polygonGraphicsOverlay
+    }
+    
+    func makeCurveGraphicsOverlay() -> AGSGraphicsOverlay {
+        // Create a simple fill symbol with outline.
+        let lineSymbol = AGSSimpleLineSymbol(style: .solid, color: .black, width: 1)
+        let filledSymbolWithCurve = AGSSimpleFillSymbol(style: .solid, color: .red, outline: lineSymbol)
+        // Create a graphics overlay for the polygons with curve segments.
+        let curveGraphicsOverlay = AGSGraphicsOverlay()
+        // Create and assign a simple renderer to the graphics overlay.
+        curveGraphicsOverlay.renderer = AGSSimpleRenderer(symbol: filledSymbolWithCurve)
+        return curveGraphicsOverlay
+    }
+    
+    /// Create a heart-shape geometry with Bezier and elliptic arc segments.
+    ///
+    /// - Parameters:
+    ///   - center: The center of the square that contains the heart shape.
+    ///   - sideLength: The side length of the square.
+    /// - Returns: A heart-shape geometry.
+    func makeHeartGeometry(center: AGSPoint, sideLength: Double) -> AGSGeometry? {
+        guard sideLength > 0 else { return nil }
+        let sr = center.spatialReference
+        // The x and y coordinates to simplify the calculation.
+        let minX = center.x - 0.5 * sideLength
+        let minY = center.y - 0.5 * sideLength
+        // The radius of the arcs.
+        let arcRadius = sideLength * 0.25
+        
+        // Bottom left curve.
+        let leftCurveStart = AGSPoint(x: center.x, y: minY, spatialReference: sr)
```

Comment:
Since you've updated the other variable names, I recommend `sr` be changed as well.